src/snowflake/snowpark/dataframe.py

```diff
@@ -3771,6 +3771,16 @@ def convert(col: ColumnOrName) -> Expression:
         exprs = [convert(col) for col in parse_positional_args_to_list(*cols)]
         return exprs
 
+    def printSchema(self) -> str:
+        return "root\n%s" % (
```

Comment:
NIT: use f-strings instead?

Reply:
I would second this. Looking at the failing tests, looks like we missed a `.format()`. We are moving away from `.format()` in favor of f-strings